cardano-cli/test/cardano-cli-golden/Test/Golden/Babbage/Governance/AnswerPoll.hs

```diff
@@ -1,6 +1,6 @@
 {-# LANGUAGE OverloadedStrings #-}
 
-module Test.Golden.Shelley.Governance.AnswerPoll where
+module Test.Golden.Babbage.Governance.AnswerPoll where
```

Comment:
When in era-based, we shouldn't organise by era because it can get complicated.

Maybe the module should be just `Test.Golden.EraBased.Governance.AnswerPoll` to say we are testing era-based commands.

Reply:
OK I see :+1:

I changed to `Test.Golden.EraBased.Governance.AnswerPoll`.